src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java

```diff
@@ -46,18 +51,28 @@ public void ImageStorage_001_deleteImage() {
 
     DeletePageObject delete = file.deletePage();
     delete.submitDeletion();
-    Assertion.assertTrue(filesPage.getBannerNotifications().isNotificationMessageVisible(),
+
+    List<Notification> confirmNotifications = filesPage.getNotifications(CONFIRM_NOTIFICATION);
+    Assertion.assertTrue(confirmNotifications.size()==1,
+            "Number of banner notifications is invalid");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Number of banner notifications is invalid" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)